feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -651,8 +661,18 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []*oc.Co
 			if p.fwVerValidation {
 				fwVer := card.GetFirmwareVersion()
 				t.Logf("Component %s FirmwareVersion: %s", cName, fwVer)
+
+				isTransceiver := card.GetType() == componentType["Transceiver"]
+				is400G := false
+				if isTransceiver {
+					is400G = strings.Contains(card.GetTransceiver().GetEthernetPmd().String(), "ETH_400GBASE")
```

Comment:
```suggestion
					is400G = strings.Contains(card.GetTransceiver().GetEthernetPmd().String(), "ETH_400GBASE_ZR")
```

Reply:
https://github.com/openconfig/public/blob/5d38d8531ef9c5b998262207eb6dbdae8968f9fe/release/models/optical-transport/openconfig-transport-types.yang#L1112C12-L1115

Reply:
I think the requirement is for 400ZR only.  Can you confirm Xulin?

Reply:
After checking with our SME for optical transceivers, we expect all 400G transceivers to expose a firmware version.